exercises/index.html

```diff
@@ -12,7 +12,9 @@
 </head>
 
 <body>
-  <div id="app"></div>
+  <div id="app">
+    asd
```

Comment:
@varkrish - this seems like a typo? I guess it can be removed....

Reply:
Oops.  Should look good now.